### PR TITLE
Add links to cacheing blog posts

### PR DESCRIPTION
**Pull request by abhi18av**
Hi folks 👋 

I thought that [Resume troubleshooting](https://training.nextflow.io/basic_training/cache_and_resume/#resume-troubleshooting) might be a good place to mention the three cacheing related blogposts.

I was in doubt whether it should be at the end of the content as another point but then I thought, for the curious (or frustrated) folks putting it as a tip might be reasonable.

@mribeirodantas , desculpa que não adiocionei o português - nem tinha certeza que escolhi o lugar certo para adicionar o TIP em inglês. Mas posso fazer isso se vocês concordam mencionar os blogs como um TIP.

### CODE REVIEW

File: docs/basic_training/cache_and_resume.md [hunk 1]
```diff
@@ -200,6 +200,10 @@ Finally, the `-t` option enables the creation of a basic custom provenance repor
 
 ## Resume troubleshooting
 
+!!! tip
```
Comment (ewels) [inline docs/basic_training/cache_and_resume.md:203]: I would drop this below the intro paragraph, and reword / rewrite the intro paragraph a little.

Also, may as well use a numbered list - easier to read than having them inline text.

**Comment by mribeirodantas** [comment]
Don't worry, @abhi18av. You did it the right way. First in English, and once it's reviewed, the corresponding change in the Portuguese file. They need to be in separate commits so that the localization tracking (Git Localize) system can be sure they're up to date 😄

File: docs/basic_training/cache_and_resume.md (hunk 1 above)
Comment (mribeirodantas) [reply docs/basic_training/cache_and_resume.md:203]: I agree with Phil.

File: docs/basic_training/cache_and_resume.md [hunk 2]
```diff
@@ -200,6 +200,10 @@ Finally, the `-t` option enables the creation of a basic custom provenance repor
...
+
+    To learn more details about the resume mechanism and how to troubleshoot please refer the following three blog posts (i) [Demystefying Nextflow resume](https://www.nextflow.io/blog/2019/demystifying-nextflow-resume.html), (ii) [Troubleshooting Nextflow resume](https://www.nextflow.io/blog/2019/troubleshooting-nextflow-resume.html) and (iii) [Analyzing caching behavior of pipelines](https://nextflow.io/blog/2022/caching-behavior-analysis.html)
+    
 If your workflow execution is not resumed as expected with one or more tasks being unexpectedly re-executed each time, these may be the most likely causes:
```
Comment (ewels) [inline docs/basic_training/cache_and_resume.md:207]:
```suggestion
Being able to resume workflows is a key feature of Nextflow, but it doesn't always work as you expect. In this section we go through a few common reasons why Nextflow may be ignoring your cached results.

!!! tip

    To learn more details about the resume mechanism and how to troubleshoot please refer the following three blog posts:
    
     1. [Demystefying Nextflow resume](https://www.nextflow.io/blog/2019/demystifying-nextflow-resume.html)
     2. [Troubleshooting Nextflow resume](https://www.nextflow.io/blog/2019/troubleshooting-nextflow-resume.html)
     3. [Analyzing caching behavior of pipelines](https://nextflow.io/blog/2022/caching-behavior-analysis.html)
    

```

**Review by mribeirodantas — APPROVED** [review]

**Comment by abhi18av** [comment]
Thanks for the quick merge Phil and Marcel 💚

**Pull request merged**